.github/workflows/workflow-executor.yaml

```diff
@@ -109,6 +109,9 @@ on:
       speakeasy_api_key:
         description: The API key to use to authenticate the Speakeasy CLI
         required: true
+      action_pat:
+        description: A specific Github PAT used to create Pull Requests
+        required: false
```

Comment:
Wouldn't they just provide this in `github_access_token` ? We should be able to tell what type of token it is from the token itself

Reply:
Because they will need to setup way more permissions then. Replacing GITHUB_TOKEN entirely is unnecessary and just makes it way more likely something gets messed up in the set up process.